doc/ld65.sgml

```diff
@@ -298,6 +299,12 @@ Here is a description of all of the command-line options:
   information generation is currently being developed, so the format of the
   file and its contents are subject to change without further notice.
 
+  <label id="option--large-alignment">
+  <tag><tt>--large-alignment</tt></tag>
+
+  Disable warnings about a large combined alignment. See the discussion of the
+  <tt><ref id=".ALIGN" name=".ALIGN"></tt> directive for futher information.
```

Comment:
That line refers to a label in a different document (`ca65.sgml`).  It can't be linked automatically.  You must replace it with a line such as this:

<tt/.ALIGN/ directive in the ca65 document, for further information.

Reply:
Good feedback. Will make the change.

Reply:
Removed link and changed text.